lib/ProductOpener/APITest.pm

```diff
@@ -100,21 +101,8 @@ Call API to create a user
 =cut
 
 sub create_user ($ua, $args_ref) {
-	my %fields = (
-		email => 'test@test.com',
-		userid => "test",
-		name => "Test",
-		password => "testtest",
-		confirm_password => "testtest",
-		pro_checkbox => 0,
-		requested_org => "",
-		team_1 => "",
-		team_2 => "",
-		team_3 => "",
-		action => "process",
-		type => "add",
-		".submit" => "Register"
-	);
+
+	my %fields = (%default_user_form, (userid => "tea", email => 'tea@boba.com'));
```

Comment:
Hum maybe you are just testing, but this was not meant to be done here, but outside the create_user function.

The idea is that create_user should really only receive data, The "apply overrides" part below should also be removed.

It's the caller who should construct its arg by attending at the provide default hash map.

```perl
# in create_product.t

my %admin_user_args = (%default_user_form, (userid => "stephane", email => 'stephane@test.com'))
create_user($ua, \%admin_user_args)
```

This is far more flexible, for we maybe could also provides other default hashmap, or hashmap that can be combined.

Eg:

```perl
my %admin_user_args = (%default_user_form, %search_preferences_user_args, (name => "Paul"))
create_user($ua, \%admin_user_args)
```